portfolio/src/main/java/com/google/sps/servlets/DeleteServerlet.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Get comments data from Datastore. Delete all of the comments obtained
+ * from Datastore. Then, return an empty reponse.
+ */
+@WebServlet("/delete-comments")
+public class DeleteServerlet extends HttpServlet {
```

Comment:
DeleteServlet? I'm not sure whether there is a naming convention for subclasses but Serverlet sounds weird to me?

Reply:
The file that was originally provided was named DataServerlet.java so I have been following that naming convention. I can change it to DeleteComments if that is better?

Reply:
Can you point the doc to me?

I believe it's DataServlet unless I'm looking at a wrong place? https://github.com/googleinterns/step/blob/master/walkthroughs/week-3-server/step-1-servlets-walkthrough.md#your-turn

Reply:
Oh, you're right. Sorry, I've been reading it as serverlet instead of servlet by accident. I will change it.